src/LobbyForm.tsx

```diff
@@ -13,35 +15,52 @@ function LobbyForm({ socket }: Props): React.ReactElement {
   return (
     <div className={css(hostStyles.form)}>
       <Host socket={socket} />
-      <div className={css(hostStyles.or)}>- or -</div>
+      <div className={css(hostStyles.or)}></div>

```

Comment:
is this div still necessary?

Reply:
Currently left this div in to create spacing between the two components. I will look into a better way to create that space in a little bit.